src/test/java/org/openrefine/extensions/commons/importer/FileRecordToRowsTest.java

```diff
@@ -28,10 +28,10 @@ public void testGetNextRowOfCells() throws Exception {
         rows.add(frtr.getNextRowOfCells());
         rows.add(frtr.getNextRowOfCells());
 
-        Assert.assertEquals(rows.get(0), Arrays.asList("File:LasTres.jpg", "M127722", "Category:Costa Rica"));
+        Assert.assertEquals(rows.get(0).toString(), Arrays.asList("File:LasTres.jpg, M127722, Category:Costa Rica").toString());
```

Comment:
here you could also try to check that the `Recon` object is present as well (but not a big deal without)

Reply:
This check has been implemented in PR #77